AdyenActions/Components/QRCode/QRCodeActionComponent.swift

```diff
@@ -51,15 +53,17 @@ public final class QRCodeActionComponent: ActionComponent, Cancellable {
     
     /// The QR code component configurations.
     public var configuration: Configuration
-    
-    private let pollingComponentBuilder: AnyPollingHandlerProvider
+
+    private let pollingComponentBuilder: AnyPollingHandlerProvider?
     
     private var pollingComponent: AnyPollingHandler?
-    
+
     private var timeoutTimer: ExpirationTimer?
     
     private let progress = Progress()
     
+    private var qrCodeAction: QRCodeAction?
```

Comment:
instead of keeping it as an instance property, maybe pass it as parameter to the functions that needed?

Reply:
sure.